File: server_mybatis/fli/src/main/java/com/leepark/fli/model/service/PlaylistContentService.java
package com.leepark.fli.model.service;

import com.leepark.fli.model.dto.PlaylistContent;

// Service -> DAO의 DB 접근 기본 메서드를 조립해서 새로운 기능을 구현하는 것
public interface PlaylistContentService {
	// 유저 플레이리스트에 노래 추가
    Integer insertMusic(PlaylistContent playlistContent);
    Integer deleteMusic(PlaylistContent playlistContent);
    Integer deleteAll(Integer playlistId);
}

File: client/fli/app/src/main/java/com/ssafy/fli/network/PlayListContentService.kt
package com.ssafy.fli.network

import com.ssafy.fli.model.PlayListContent
import retrofit2.http.Body
import retrofit2.http.HTTP
import retrofit2.http.POST

interface PlayListContentService {
    @POST("playlistContent/insertMusic")
    suspend fun insertPlayListContent(@Body playListContent: PlayListContent): Int

    @HTTP(method = "DELETE", path = "playlistContent/deleteMusic", hasBody = true)
    suspend fun deletePlayListContent(@Body playlistContent: PlayListContent): Int
}